website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -50,6 +50,8 @@ account provided to pods if no service account is defined.
 
 ### Setting `issuer` for Kubernetes authentication
 
+-> **Deprecated:** The `issuer` parameter has been deprecated as of Vault 1.9 and will be removed in a future release.
```

Comment:
In line 55 we are saying that issuer more than likely needs to be configured. Maybe add something stating that there is no need to configure the issuer if running vault-1.9+

Reply:
1ff8749 - States that if you are running a version prior to 1.9 then you will need to configure the issuer.